paddle/framework/selected_rows.cc

```diff
@@ -12,5 +12,48 @@ limitations under the License. */
 #include "paddle/framework/selected_rows.h"
 
 namespace paddle {
-namespace framework {}  // namespace framework
+namespace framework {
+void SerializeToStream(std::ostream& os, const SelectedRows& selected_rows,
+                       const platform::DeviceContext& dev_ctx) {
+  SerializeToStream(os, selected_rows.value(), dev_ctx);
```

Comment:
How about putting this line after height serialized? Tensor size maybe large.

Reply:
The first field in `Tensor::SerializeToStream(...)` is version,  if putting height in front of `Tensor::SerializeToStream(...)`, maybe we could add a version for `SelectedRows`.

Reply:
Done.